extra-dev/packages/coq-file-sync/coq-file-sync.dev/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+maintainer: "Yishuai Li <yishuai@cis.upenn.edu>"
+
+homepage: "https://github.com/liyishuai/file-sync"
+dev-repo: "git+https://github.com/liyishuai/file-sync.git"
+bug-reports: "https://github.com/liyishuai/file-sync/issues"
+license: "MPL-2.0"
+
+synopsis: "File synchroniser specification in Coq"
+description: """
+Coq formalisation of the Unison file synchroniser"""
+
+build: [make "-j%{jobs}%"]
+run-test: [make "-j%{jobs}%" "test"]
```

Comment:
```suggestion
```
If you remove the test dependency, you will probably have failing test unless we also do this.